synapse/lib/types.py

```diff
@@ -934,6 +935,22 @@ def _normPyTuple(self, valu):
     def indx(self, norm):
         return s_common.buid(norm)
 
+    def repr(self, norm, defval=None):
+        (n1form, n1valu), (n2form, n2valu) = norm
+        try:
+            n1repr = self.modl.type(n1form).repr(n1valu, n1valu)
+            n2repr = self.modl.type(n2form).repr(n2valu, n2valu)
+            # We are doing some arbitrary storm escaping here to make things safe.
+            # If we ever make repr behavior more universally storm safe, we will
+            # need to change calling this ourselves.
+            s = f'(({n1form}, "{s_chop.stormstring(n1repr)}"), ({n2form}, "{s_chop.stormstring(n2repr)}"))'
+            return s
+        except asyncio.CancelledError:
```

Comment:
there's nothing in this code path that calls anything async. This is unnecessary

Reply:
habit :D